lte/protos/pipelined.proto

```diff
@@ -198,6 +198,217 @@ message PacketDropTableId {
   int32 table_id = 1;
 }
 
+//-------------------------------------------------------------------------
+// SMF to UPF Proto
+//-------------------------------------------------------------------------
+
+
+
+message NodeID {
+  enum NodeIDType {
+    IPv4 = 0;
+    IPv6 = 1;
+    FQDN = 2;
+    }
+  NodeIDType node_id_type = 1;
+  string node_id = 2;
+}
+
+
+//CPFseid message
+message CpFSeid {
+  uint64 f_seid = 1;
+  NodeID node_id = 2;
+}
+
+message OuterHeadRemoval {                //conditional PDR Message Structures
+  uint32 o_hd_remo_descr = 1;
+}
+
+
+//message LocalFTeid {                      //not sure about the message structure
+ ///local_f_teid = ;                        //doubt
+//}
+
+
+
+message SdfFilters {                               //8.2.5
+
+  FlowDescriptor fd = 1;
+  bytes ttc = 2;
+  bytes spi = 3;
+  bytes fl = 4;
+  uint32 sdf_id = 5;
+}
+
+message FlowDescriptor {                //include
+FlowMatchNew match = 1;
+  enum Action {
+    PERMIT = 0;
+    DENY = 1;
+  }
+  Action action = 2;
+}
+
+message FlowMatchNew {
+  string ipv4_src = 1;
+  string ipv4_dst = 2;
+  uint32 tcp_src = 3;
+  uint32 tcp_dst = 4;
+  uint32 udp_src = 5;
+  uint32 udp_dst = 6;
+}
+
+message MacAddress {            //8.2.93
+  bytes mac_src_value = 1;
+  bytes mac_dst_value = 2;
+  bytes upper_mac_src_value = 3;
+  bytes upper_mac_dst_value = 4;
+}
+
+
+//Table 7.5.2.2-3: Ethernet Packet Filter IE within PFCP Session Establishment Request
+message EthPackFilter {
+  uint32 eth_filt_id = 1;
+  uint32 eth_filt_prop = 2;
+  MacAddress mac_addr = 3;
+  uint32 c_tag = 4;                                   //not verified the tag values
+  uint32 s_tag = 5;
+  SdfFilters sdf_filters = 6;
+  uint32 ethertype = 7;
+}
```

Comment:
If we going to support ethernet filtering, we need to support otherL2 related stuff in session establishment (5.13.1).
I do not see that in this this protobuf, Am I missing something?